pages/index.js

```diff
@@ -1,12 +1,14 @@
 import { styled } from "styled-components";
 import { travel } from "@/data/data";
 import TripItem from "@/components/TripItem";
+import NewTripButton from "@/components/CreateTripButton";
 
 export default function ListOverview({ name, destination }) {
   return (
     <FlexContainer>
       <h2>Upcoming trips</h2>
       <TripItem />
+      <NewTripButton>Add new trip</NewTripButton>
```

Comment:
I would suggest to name your buttons not after the functionality but after the look, then you can easily reuse them. for example:
- DeleteButton becomes `<FilledButton variant='danger'>`
- SubmitButton becomes `<FilledButton variant='success'>`
- BackButton becomes `<CircularButton>`

By this your components can be used in various ways, it would be strange to use the "NewTripButton" as your submit button for example, but you can use the "FilledButton" in both cases.

Reply:
Hey Felix, that very good! i was struggling with a proper naming :) thanks